slide-deck.js

```diff
@@ -1,6 +1,7 @@
+// @ts-check
 class slideDeck extends HTMLElement {
   // template
-  static appendShadowTemplate = (node) => {
+  #appendShadowTemplate = () => {
```

Comment:
Makes sense for this to be private. But doesn't it seem like it should still be static?